week08_pomdp/practice_pytorch.ipynb

```diff
@@ -256,19 +255,36 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "def evaluate(agent, env, n_games=1):\n",
+    "def evaluate(agent,\n",
+    "             env,\n",
+    "             video_dir: Union[Text, None]=None,\n",
+    "             n_games=1,\n",
+    "             record: Union[bool, Text]=False,\n",
+    "             debug: bool=False):\n",
+    "  \n",
     "    \"\"\"Plays an entire game start to end, returns session rewards.\"\"\"\n",
-    "\n",
+    "    # For Gym Debugging\n",
+    "    gym.logger.set_level(gym.logger.DEBUG) if debug else gym.logger.set_level(gym.logger.DISABLED)\n",
+    "    \n",
+    "    # For Recording\n",
+    "    if record:\n",
+    "      if record ==\"Statistic\":\n",
```

Comment:
This is strange semantics.

If `record is False`, no recording happens. If `record == 'Statistic'`, it starts recording with this new wrapper. However, if `record` is any other string (or `True`), it starts recording without this wrapper.

I'd suggest just passing in two flags like `record: bool` and `record_statistics: bool`. Or use something like `record: Optional[str]` and support 3 values: `None`, `'video'`, `'video_with_statistics'`.